inst/safetyGraphics_app/server.R

```diff
@@ -83,32 +83,13 @@ observeEvent(settings_new$status(),{
     }
   })
 
-
-
-
-#
-#     callModule(
-#       module = render_edish_chart,
-#       id = paste0("chart", "edish"),
-#       data = reactive(dataUpload_out$data_selected()),
-#       settings = reactive(settings_new$settings()),
-#       valid = reactive(settings_new$status()[["edish"]]$valid)
-#     )
-#     callModule(
-#       module = render_safetyhistogram_chart,
-#       id = paste0("chart", "safetyhistogram"),
-#       data = reactive(dataUpload_out$data_selected()),
-#       settings = reactive(settings_new$settings()),
-#       valid = reactive(settings_new$status()[["safetyhistogram"]]$valid)
-#     )
-
 for(chart in all_charts){
   callModule(
     module = renderChart,
     id = paste0("chart", chart),
     data = reactive(dataUpload_out$data_selected()),
     settings = reactive(settings_new$settings()),
-    valid = reactive(settings_new$status()$valid),
+    valid = reactive(settings_new$status()$charts[[chart]]),
```

Comment:
is this loop working ok?  this type of code was very buggy for me before.

Reply:
Seems ok so far. Will keep an eye out for issues.